easybuild/easyconfigs/a/ant/ant-1.9.6-Java-1.8.0_74.eb

```diff
@@ -0,0 +1,26 @@
+name = 'ant'
+version = '1.9.6'
+
+homepage = 'http://ant.apache.org/'
+description = """Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files
+ as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['apache-%(name)s-%(version)s-src.tar.gz']
+source_urls = ['http://archive.apache.org/dist/%(name)s/source/']
+
+java = 'Java'
+javaver = '1.8.0_74'
+versionsuffix = '-%s-%s' % (java, javaver)
```

Comment:
In 2.7.0 we introduced a couple of new string templates to make this easier:
```
    %(javashortver)s: short version for Java (<major>.<minor>)
    %(javaver)s: full version for Java
```
Just load `Java` under dependencies and then this templates will work.